modules/ossm-installing-operator.adoc

```diff
@@ -4,13 +4,17 @@
 :_mod-docs-content-type: PROCEDURE
 [id="ossm-installing-operator_{context}"]
 = Installing the {SMProductShortName} Operator
-:context: ossm-installing-operator
+
+[WARNING]
+====
+For clusters without {SMProduct} instances, install the {SMProductShortName} Operator. {SMProduct} operates cluster-wide and needs a scope configuration to prevent conflicts between {istio} control planes. For clusters with {SMProduct} 3 or later, see xref:../install/ossm-deploying-multiple-service-meshes-on-single-cluster.adoc#ossm-deploying-multiple-service-meshes-on-single-cluster[Deploying multiple service meshes on a single cluster].
```

Comment:
We can't use xrefs in modules. In this case, you would need to remove the xref, and instead link to this section by using an additional resources list underneath the place where this module is included in the assembly.
```suggestion
For clusters without {SMProduct} instances, install the {SMProductShortName} Operator. {SMProduct} operates cluster-wide and needs a scope configuration to prevent conflicts between {istio} control planes. For clusters with {SMProduct} 3 or later, see "Deploying multiple service meshes on a single cluster".
```